node/node.go

```diff
@@ -460,6 +460,11 @@ func (node *AlgorandFullNode) Stop() {
 	}
 }
 
+func (node *AlgorandFullNode) Shutdown() {
```

Comment:
how is it different from Stop() ? mayb move these close statements there?

Reply:
```
	registry, err := ensureParticipationDB(genesisDir, node.log)
```
opened in `MakeFull` and not reopened in `Start`,  so closing in `Shutdown`:
```
	node.accountManager.Registry().Close()`
```

Same for 
```
	crashAccess, err := db.MakeAccessor(crashPathname, false, false)
```
opened in `MakeFull` and not reopened in `Start`,  so closing in `Shutdown`:
```
	node.agreementService.Accessor.Close()
```

Reply:
then Shutdown should be called `daemon/algod/server.go`'s `Server::Stop`